testsuite/gnat2goto/tests/deferred_constant/test.adb

```diff
@@ -0,0 +1,7 @@
+with Deferred_Const; use Deferred_Const;
```

Comment:
?

Reply:
I am not sure what this test is checking.  Originally I provide a test program called use_deferred_const

with Deferred_Const; use Deferred_Const;
procedure Use_Deferred_Const is
   No_Use : Priv := Def_Const;
begin
   Inc (No_Use);
end Use_Deferred_Const;

If we add another priv constant to the package Deferred_Const as below:

package Deferred_Const is
   type Priv is private;
   
   Def_Const : constant Priv;
   Def_Const_Plus_1 : constant Priv;
   
   procedure Inc (P : in out Priv);
   
private
   type Priv is range 1 .. 10;
   
   Def_Const : constant Priv := 1;
   Def_Const_Plus_1 := Def_Const + 1;
end Deferred_Const;

Then in Use_Deferred_Constant after the call to Inc we could place the
pragma Assert (No_Use = Def_Const_Plus_1);

Which should succeed.

Reply:
I haven't added the test, I just wanted it to reach symex. This way the the range-check for `Def_Const` is included in the verification. Even if we don't actually refer to anything from the package.